tensorflow_addons/optimizers/tests/lookahead_test.py

```diff
@@ -120,6 +121,30 @@ def test_fit_simple_linear_model():
     assert max_abs_diff < 1e-3
 
 
+def test_fit_simple_linear_model_mixed_precision():
+    np.random.seed(0x2019)
+    tf.random.set_seed(0x2019)
+
+    x = np.random.standard_normal((10000, 3))
+    w = np.random.standard_normal((3, 1))
+    y = np.dot(x, w) + np.random.standard_normal((10000, 1)) * 1e-4
+
+    with policy.policy_scope(policy.Policy("mixed_float16")):
+        model = tf.keras.models.Sequential()
+        model.add(tf.keras.layers.Dense(input_shape=(3,), units=1))
+        model.compile(Lookahead("sgd"), loss="mse")
+
+    model.fit(x, y, epochs=3)
+
+    x = np.random.standard_normal((100, 3))
+    y = np.dot(x, w)
+    predicted = model.predict(x)
+
+    max_abs_diff = np.max(np.abs(predicted - y))
+    assert max_abs_diff < 2.3e-3
+    assert max_abs_diff >= 1e-3
```

Comment:
Why assert a greater than difference here? Mixed precision variation?

Reply:
It Is what I think. There Is a default dynamic loss scaling to avoid numerical precision underrun https://www.tensorflow.org/api_docs/python/tf/keras/mixed_precision/experimental/Policy?version=nightly